libs/velo-external-db-core/src/service/data.ts

```diff
@@ -32,37 +31,51 @@ export default class DataService {
 
     async insert(collectionName: string, item: Item, fields?: ResponseField[]) {
         const resp = await this.bulkInsert(collectionName, [item], fields)
-        return { item: asWixData(resp.items[0]) }
+        return { item: resp.items[0] }
```

Comment:
there is still need in insert/update/delete endpoints? we are using them anywhere?

Reply:
`schemaAware` uses insert/update/remove, but indeed the router uses the bulk of each endpoint, so there is actually no practical use for these functions... but I still don't see the need to delete them